{"name": "test_firstTest", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_myFirstTest.TestMush object at 0x105f48980>\n\n    def test_firstTest(self):\n        print('First test1')\n        print('Second test')\n>       assert False\nE       assert False\n\nsrc/test_myFirstTest.py:5: AssertionError"}, "start": 1745222944569, "stop": 1745222944569, "uuid": "edcc012f-108a-49e1-b2e7-7c6a02db538d", "historyId": "98b9d2a354c1b40711ae7fd21ed656bb", "testCaseId": "98b9d2a354c1b40711ae7fd21ed656bb", "fullName": "src.test_myFirstTest.TestMush#test_firstTest", "labels": [{"name": "parentSuite", "value": "src"}, {"name": "suite", "value": "test_myFirstTest"}, {"name": "subSuite", "value": "TestMush"}, {"name": "host", "value": "Musheer-Khan.local"}, {"name": "thread", "value": "38114-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.test_myFirstTest"}]}